src/site/paragraphs/table.drupal.liquid

```diff
@@ -33,28 +33,30 @@
         {% endif %}
         {% assign colLabels = entity.fieldTable.value.0  %}
         {% for value in entity.fieldTable.value %}
-            {% if forloop.first == true %}
-                <thead>
-                    <tr role="row">
-                        {% for column in value %}
-                            <th role="columnheader" scope="col">{{ column }}</th>
-                        {% endfor %}
-                    </tr>
-                </thead>
-                <tbody>
-            <!-- this elsif is here to prevent the caption from showing up as a row -->
-            {% elsif forloop.last == false %}
-                    <tr class="vads-u-padding-top--5" role="row">
-                        {% for column in value %}
-                                <td class="vads-u-padding-bottom--0 column-label vads-u-font-weight--bold medium-screen:vads-u-display--none" role="cell">
-                                    <dfn> {{colLabels | getValueFromObjPath: forloop.index0}}      </dfn>
-                                </td>
-                                <td class="column-value"  role="cell">
-                                    <dfn class="vads-u-display--block"> {{ column }}  </dfn>
-                                </td>
-                        {% endfor %}
-                    </tr>
-            {% endif %}
-                </tbody>
+          {% if forloop.first == true %}
+            <thead>
+              <tr role="row">
+                {% for column in value %}
+                  <th role="columnheader" scope="col">{{ column }}</th>
+                {% endfor %}
+              </tr>
+            </thead>
+            <tbody>
+          {% else %}
+            <tr class="vads-u-padding-top--5" role="row">
+              {% for column in value %}
+                  <td class="vads-u-padding-bottom--0 column-label vads-u-font-weight--bold medium-screen:vads-u-display--none" role="cell">
+                      <dfn> {{colLabels | getValueFromObjPath: forloop.index0}}      </dfn>
```

Comment:
Just checking the spacing here

Reply:
Dan we usually use 4 spaces? Man that is beefy whitespace

Reply:
haha we usually use 2 spaces i think! 🙂 and ya agreed 😂 this could probably be updated to:

```liquid
<dfn>{{ colLabels | getValueFromObjPath: forloop.index0 }}</dfn>
```

if we wanted!